test/Elastica/Aggregation/GeoBoundsTest.php

```diff
@@ -8,7 +8,7 @@
 
 class GeoBoundsTest extends BaseAggregationTest
 {
-    protected function _getIndexForTest()
+    private function getIndexForTest()
```

Comment:
The reason I kept the `_` is not because of the modifiers, they exist for quite some time. I kept it for humans as it makes it very easy to see if a method is public or not. Similar to Golang that uses upper and lower case for it. As I said, I still like the `_` but I think also it makes sense to follow what other large projects are doing to have a "unified" code base.

What are others doing?